cassandra-4/src/test/resources/docker/cassandra.yaml

```diff
@@ -222,7 +222,7 @@ cdc_enabled: true
 # separate spindle than the data directories. If not set, the default directory is
 # $CASSANDRA_HOME/data/cdc_raw.
 # cdc_raw_directory: /var/lib/cassandra/cdc_raw
-cdc_raw_directory: /var/lib/cassandra/cdc_raw_directory
+cdc_raw_directory: /var/lib/cassandra/cdc_raw
```

Comment:
just double checking, is this what we want? (in case it was a typo)

Reply:
Yes, this was intentional.